README.md

```diff
@@ -132,6 +132,12 @@ export LD_LIBRARY_PATH=/<YOUR_PRODUCER_SDK_CPP_DOWNLOAD>/amazon-kinesis-video-st
 This should resolve native library loading issues.
 
 ## Release Notes
+
+### Release 2.0.0 (27 May 2020)
+* Migration to new CPP Producer SDK base
+* Updated docker scripts to incorporate the changes in producer SDK
+* Provision of CachedInfoMultiAuthServiceCallbacks callback to use different credential providers for different streams in one client with existing stream information
```

Comment:
@disa6302 , can you point me to the code/PR which added Multi Auth Callback? I would like to understand it better.

Reply:
@sharadrj , 

This is the one: 
https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-java/pull/88